src/components/TypesOfRecipeCards/ProfilePost.jsx

```diff
@@ -0,0 +1,92 @@
+import { PushPin } from "@phosphor-icons/react";
+import { Link } from "react-router-dom";
+import { pinWhite, pinSize } from "../../shared/style";
+
+const ProfilePost = ({ pinned, post, reviewer }) => {
+  //TO DO - need to add function to pin recipe
+
+  const isRecipe = post && post.recipeId ? false : true;
```

Comment:
I think this is more confusing than before. This is not what I meant when I said a different method. Ideally I don't think you should use recipeId. If it's not possible, please add a comment to explain because it's not intuitive.